app/controllers/users_controller.rb

```diff
@@ -15,6 +16,16 @@ def update
     end
   end
 
+  def delete_card
+    customer = Stripe::Customer.retrieve(params[:customer_id])
+    customer.sources.retrieve(params[:card_id]).delete
```

Comment:
I think it's best to delete the user from stripe too. But when we delete the user from stripe we still have access to any payments he has made so we have a record of that.